tests/bindings-test/ProtocolTest.cs

```diff
@@ -393,7 +393,7 @@ public override bool Equals (object obj)
 
 			public override int GetHashCode ()
 			{
-				return HashCode.Combine(Name, Value);
+				return HashCode.Combine (Name, Value);
```

Comment:
@rolfbjarne the code formatter fixed files I didn't touch :(

Reply:
Yea, this is a known issue. I've fixed it by merging main into your branch.

Reply:
Ah ok. I did a rebase this morning, hopefully that won't reverse things 👍